db/sqlc/academic_years_test.go

```diff
@@ -0,0 +1,71 @@
+package db
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+
+	"github.com/VinCPR/backend/util"
+	"github.com/stretchr/testify/require"
+)
+
+func createRandomAcademicYear(t *testing.T) AcademicYear {
+	RandDate := util.RandomDate()
+	arg := CreateAcademicYearParams{
+		Name:      util.RandomName(),
+		StartDate: RandDate,
+		EndDate:   RandDate.AddDate(0, 0, 7*rand.Intn(13)),
+	}
+	academic_year, err := testQueries.CreateAcademicYear(context.Background(), arg)
```

Comment:
Please use camel case instead of snake case, it is golang convention to name variable using camel case: `academicYear`